### PR TITLE
rename person to spending 

### PR DESCRIPTION
**Pull request by austinsantoso**
Rename all instance of person to spending

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any spending obtaining a copy
+Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (Nanosync) [inline LICENSE]: accidental change

File: docs/templates/LICENSE [hunk 2]
```diff
@@ -5,11 +5,11 @@ MIT License
 
 Copyright (C) 2012-2018 Dan Allen, Ryan Waldron and the Asciidoctor Project
 
-Permission is hereby granted, free of charge, to any spending obtaining a copy
+Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (Nanosync) [inline docs/templates/LICENSE]: ditto

**Review by Nanosync — CHANGES_REQUESTED** [review]
accidental

File: LICENSE (hunk 1 above)
Comment (austinsantoso) [reply LICENSE]: isn't this correct, i changed the spending to person?

File: src/main/java/seedu/address/model/spending/Spending.java [hunk 3]
```diff
@@ -61,45 +59,45 @@ public Address getAddress() {
...
+                && otherSpending.getAddress().equals(getAddress())
+                && otherSpending.getEmail().equals(getEmail())
+                && otherSpending.getTags().equals(getTags());
     }
 
     @Override
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
-        return Objects.hash(name, phone, email, address, tags);
+        return Objects.hash(name, phone, email, tags);
```
Comment (Nanosync) [inline src/main/java/seedu/address/model/spending/Spending.java]: address field accidentally removed from hash

**Review by Nanosync — CHANGES_REQUESTED** [review]
last minor change before merge

**Review by Nanosync — APPROVED** [review]
Thank you Austin!

**Comment by choongyx** [comment]
#10, #8, #9

**Pull request merged**